americangut/util.py

```diff
@@ -492,7 +492,7 @@ def clean_and_reformat_mapping(in_fp, out_fp, body_site_column_name,
             body_site = body_site.split('_', 1)[-1].replace("_", " ")
         elif body_site.startswith('UBERON:'):
             body_site = body_site.split(':', 1)[-1]
-        elif body_site in ['NA', 'unknown', '', 'no_data', 'None']:
+        elif body_site in ['NA', 'unknown', '', 'no_data', 'None', 'Unknown']:
```

Comment:
What we mean is does `Unknown` mean something different to `unknown` which means something different than `UNKNOWN`. If this is the case, this pull request is fine. If they all mean the same thing, converting to lowercase and then doing the comparison means we don't have to worry about what case the data actually is.